app/components/ny_times/client.rb

```diff
@@ -0,0 +1,28 @@
+module NyTimes
+  # HTTParty client for the NewYork Times that takes an article path
+  # and get's the text of the article
+  class Client
```

Comment:
Assumes too much for instance variable '@response_body' (InstanceVariableAssumption)